src/database/sql_database.cc

```diff
@@ -1471,16 +1459,16 @@ std::string SQLDatabase::incrementUpdateIDs(const std::unordered_set<int>& ids)
     }
     commit("incrementUpdateIDs 2");
 
-    std::unique_ptr<SQLRow> row;
     std::vector<std::string> rows;
-    while ((row = res->nextRow())) {
+    rows.reserve(res->getNumRows());
+    while (auto row = res->nextRow()) {
         std::ostringstream s;
         s << row->col(0) << ',' << row->col(1);
         rows.emplace_back(s.str());
     }
 
     if (rows.empty())
-        return "";
+        return {};
```

Comment:
I'd add a newline.

Reply:
Why a newline? We want to return an empty string (i.e. default constructed)

Reply:
No I mean between both return statements.

Reply:
Am I seeing the wrong line which you are talking about? There in only one modified return.

Reply:
```
        return {};
<-------newline here
    return join(rows, ",");
```

Reply:
I only saw that after switching to full file view. Yes, a new line would look better. I can improve this tomorrow.